pkg/conditions/conditions.go

```diff
@@ -0,0 +1,16 @@
+package conditions
+
+import (
+	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+)
+
+const (
+	// BindingReady indicates that the binding succeeded
+	BindingReady conditionsv1.ConditionType = "Binding is ready"
```

Comment:
This string is part of the API, and should be parse-able:

```suggestion
	BindingReady conditionsv1.ConditionType = "Ready"
```